lib/puppet/transport/schema/cisco_nexus.rb

```diff
@@ -0,0 +1,33 @@
+require 'puppet/resource_api'
+
+Puppet::ResourceApi.register_transport(
+  name: 'cisco_nexus',
+  desc: 'This transport connects to a Cisco Nexus Switch.',
+  connection_info: {
+    host: {
+      type: 'Optional[String]',
+      desc: 'The FQDN or IP address of the device to connect to.',
+    },
+    port: {
+      type: 'Optional[String]',
+      desc: 'The port of the device to connect to..',
+    },
+    transport: {
+      type: 'Optional[Enum["http", "https"]]',
+      desc: 'The type of transport protocol to use when connecting to the device. If not specified this will default to "http"',
+    },
+    verify_mode: {
+      type: 'Optional[Enum["peer", "client-once", "fail-no-peer", "none"]]',
+      desc: 'The type of OpenSSL client verification mode to use. Only applies if `transport` is `https`',
+    },
+    user: {
```

Comment:
Update the README and any other places where 'username' is used as an example

Reply:
Changes references o username and hot in readme